src/components/functionselect/functionselect.js

```diff
@@ -44,12 +44,15 @@ angular.module('vlui')
             'secondsmilliseconds'
           ]
         };
-        timeUnits.all = timeUnits.aboveFold.concat(timeUnits.belowFold);
+        temporalFunctions.all = _.concat(temporalFunctions.aboveFold, temporalFunctions.belowFold);
```

Comment:
In fact, `temporalFunctions.all` never get used.  I don't know why you need to declare it at all.  Why don't merge it into the logic below.